cmd/entrypoint/secrets.go

```diff
@@ -17,6 +18,7 @@ import (
 	"github.com/datawire/dlib/derror"
 	"github.com/datawire/dlib/dlog"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```

Comment:
Style: Please order the imports in groups of
```go
import (
    "stdlib"
    
    "3rd-party"
    
    "github.com/datawire/*"
)
```
(I see that the `"k8s.io/api/core/v1" there is already breaking this.)
(We really should turn that linter back on.)

Reply:
As long as there is a defined style I can stick to :slightly_smiling_face:  :+1: